### PR TITLE
Lukas/expand panden

### PR DESCRIPTION
**Pull request by arbaco**
HI @barrydaniels-nl and @jeroenriemens 

This is and expand of panden materialized views query to add array of verblijfsobjectenIdentificaties to panden endpoint.

I made changes based on @jeroenriemens suggestions.

Let me know if it is ok. 

You can test sql by `and bag_panden.identificatie =  '0363100012186092'`

Thanks in advance

### CODE REVIEW

File: datasets/dataverkenner/bagpandenm/dataset.sql
[the remark this answers is not in the document]
Comment (arbaco) [reply datasets/dataverkenner/bagpandenm/dataset.sql:1]: The `verblijfsobjecten.vots::_VARCHAR` part of the code is a type casting operation in SQL. In PostgreSQL, you can use :: to cast a value from one data type to another.
In the provided code, it's converting the vots column from the verblijfsobjecten table (or subquery in this context) to VARCHAR type. The vots column was created as an array in the earlier part of the SQL using array_agg function, this statement likely converts the array to a VARCHAR or string format.

[the remark this answers is not in the document]
Comment (jeroenriemens) [reply datasets/dataverkenner/bagpandenm/dataset.sql:1]: ok, in other cases like dataverkenner_kadastraleobjectm_kadastraleobject we don't do that conversion, so I was wondering why we would do it here

[the remark this answers is not in the document]
Comment (arbaco) [reply datasets/dataverkenner/bagpandenm/dataset.sql:1]: Hi Jereon, Yes it is also there in :
`dataverkenner_kadastraleobjectm_kadastraleobject`

https://github.com/Amsterdam/amsterdam-schema/blob/dd7000d3d32b6df42e217a42cb07c209f3625190/datasets/dataverkenner/kadastraleobjectm/dataset.sql#L40

`verblijfsobjecten.vots::_VARCHAR as "heeft_een_relatie_met_bag_verblijfsobject_identificaties",`

**Review by jeroenriemens — APPROVED** [review]
okay

**Comment by Yrob** [comment]
3 maanden geen activiteit op deze pr. Gesloten.

**Review by barrydaniels-nl — COMMENTED** [review]
OK

**Pull request closed without merging**